diesel/src/pg/expression/expression_methods.rs

```diff
@@ -916,7 +965,11 @@ pub trait PgRangeExpressionMethods: Expression + Sized {
 
     /// Creates a PostgreSQL `&<` expression.
     ///
-    /// This operator returns whether a range doesn't extend to the right of another
+    /// This operator returns true whether the argument range extend to the right of the current range
+    ///
+    /// Postgresql defines "extends" as does not have a lower bound smaller than the lower bound of the
+    /// self range. That means the right hand side range can overlap parts of the left hand side
+    /// range or be on the right side of the left hand side range
```

Comment:
A small ASCII art would be useful here and in other range operators

Reply:
That's a great idea, I updated the PR to include those examples as well.